js/ascendex.js

```diff
@@ -1703,6 +1704,112 @@ module.exports = class ascendex extends Exchange {
         return await this.v2PrivateAccountGroupGetFuturesPosition (this.extend (request, params));
     }
 
+    parseFundingRate (fundingRate, market = undefined) {
+        //
+        // {
+        //     time: "1640078764560",
+        //     symbol: "BTC-PERP",
+        //     markPrice: "48499.644083351",
+        //     indexPrice: "48530.35",
+        //     openInterest: "252.1125",
+        //     fundingRate: "-0.000631045",
+        //     nextFundingTime: "1640102400000"
+        // }
+        //
+        const marketId = this.safeString (fundingRate, 'symbol');
+        const symbol = this.safeSymbol (marketId, market);
+        const currentTime = this.safeInteger (fundingRate, 'time');
+        const nextFundingRate = this.safeNumber (fundingRate, 'fundingRate');
+        const nextFundingRateTimestamp = this.safeInteger (fundingRate, 'nextFundingTime');
+        let previousFundingTimestamp = undefined;
+        if (nextFundingRateTimestamp !== undefined) {
+            previousFundingTimestamp = nextFundingRateTimestamp - 28800000;
+        }
+        return {
+            'info': fundingRate,
```

Comment:
Can we move this to the bottom?

Reply:
Tbh, i'd rather keep on top, we found that if we move it to the bottom in all structures, beginner users get more confused with the output, thinking it's not unified and replies with exchange-specific infos.